docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
This change the Docker escape char to match Powershell's so that it makes it less tedious to in-line powershell code in a Dockerfile.